### ISSUE TITLE
nixos-22.05.538.d9794b04bff: gst-plugins-good-1.20.1 fails to build with 'moc-qt5 moc' not found

### ISSUE THREAD

**Issue by unclechu** [NixOS/nixpkgs#176482]
### Steps To Reproduce
1. Have `gst_all_1.gst-plugins-good` somewhere as a dependency
2. `sudo nixos-rebuild build`

### Build log
```
Program moc-qt5 moc found: NO

ext/qt/meson.build:51:2: ERROR: Program 'moc-qt5 moc' not found or not executable

A full log can be found at /build/gst-plugins-good-1.20.1/build/meson-logs/meson-log.txt

...
error: builder for '/nix/store/ma4vdd9cy1i3amh0jwnvrlgrphka0qfd-gst-plugins-good-1.20.1.drv' failed with exit code 1
error: 1 dependencies of derivation '/nix/store/z90mxbfv1mk1h15jcny2pmzk8k3a2qsg-nheko-0.9.3.drv' failed to build
...
```

### Notify maintainers

@matthewbauer 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.18.0, NixOS, 22.05 (Quokka), 22.05.379.08950a6e29c`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.1`
 - channels(root): `"nixos, nixos-unstable"`
 - channels(wenzel): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by jtojnar** [comment]
Looks like `nheko` wants `(gst-plugins-good.override { qt5Support = true; })`, should be fixed by 985adfdd50ef7cb18b20c5a1cdb20e851fb9f56d.

**Comment by unclechu** [comment]
Now fixed in stable NixOS 22.05 channel.
